e2e/gui/src/test/java/com/epam/pipeline/autotests/utils/Utils.java

```diff
@@ -394,4 +395,12 @@ public static String entityIDfromURL() {
         String url = WebDriverRunner.getWebDriver().getCurrentUrl();
         return url.substring(url.lastIndexOf("/") + 1);
     }
+
+    public static File getFile(String filename) {
```

Comment:
Could you update SamplesMetadataTest and ObjectMetadataFileTest according to this method?